StaticAnalyzer/views/android/icon_analysis.py

```diff
@@ -112,6 +112,11 @@ def get_icon(apk_path, res_dir):
         a = apk.APK(apk_path)
         icon_resolution = 0xFFFE - 1
         icon_name = a.get_app_icon(max_dpi=icon_resolution)
+        if icon_name is None:
+            return {
+            'path': guess_icon_path(res_dir),
+            'hidden': True
+        }

```

Comment:
Is this indentation correct @superpoussin22 ?
Did you run unit test after making the changes?

Reply:
bad copy paste which break ident , solved